src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useMemo } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { getFilteredTodos } from '../../utils/getFilteredPeople';
+import { SortLink } from '../SortLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const { slug: selectedSlug = '' } = useParams();
+  const query = searchParams.get('query') || '';
+  const sexParam = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries');
+  const sortParam = searchParams.get('sort') || '';
+  const isReversed = searchParams.get('order') === 'desc';
+
+  const visiblePeople = useMemo(
+    () => getFilteredTodos(
+      people,
+      query,
+      sexParam,
+      centuries,
+      sortParam,
+      isReversed,
+    ),
+    [searchParams],
+  );
+
+  if (!visiblePeople.length) {
+    return <p>There are no people matching the current search criteria</p>;
+  }
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Name
+              <SortLink sortBy="name" />
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Sex
+              <SortLink sortBy="sex" />
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Born
+              <SortLink sortBy="born" />
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Died
+              <SortLink sortBy="died" />
+            </span>
+          </th>
```

Comment:
`th` tag duplicates for 4 times, only text changes, we can map it (u can put all column names to enum